recipes/libiconv/all/test_package/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
 
-find_package(Iconv)
+find_package(Iconv REQUIRED)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
```

Comment:
I think the example can be converted to pure c.
https://github.com/conan-io/conan-center-index/blob/master/recipes/libiconv/all/test_package/test_package.cpp